lib/core/test/bench/db/Main.hs

```diff
@@ -279,25 +293,26 @@ mkCheckpoints numCheckpoints utxoSize =
 
     utxo i = Map.fromList $ zip
         (map fst $ mkInputs i utxoSize)
-        (mkOutputs i utxoSize)
+        (mkOutputs' i utxoSize)
```

Comment:
**Request**: Can we rename the instances of `mkOutputs'` to `mkOutputs`?

It would reduce the size of the diff.

I did a search and couldn't find any instances of `mkOutputs`, so it should be safe to rename.